docs/crd.md

```diff
@@ -103,6 +103,8 @@ Below is the list of fields in the custom resource and their description:
 
   * **savepointPath** `type:string`
     If specified, the application state will be restored from this savepoint.
+    
+    **This will only have an effect on the first deploy of a cluster. Specifying a savepoint path as an update to an already running cluster is treated as a noop.**
```

Comment:
nit:  What are your thoughts on `s/cluster/application/g` ?  The reason is the existing documentation talks in terms `application` all along and using the same term minimizes confusion.